device/samples/DeviceClientAmqpSample/Program.cs

```diff
@@ -15,6 +15,10 @@ class Program
         private const string DeviceConnectionString = "<replace>";
 
         private static int MESSAGE_COUNT = 5;
+        private static String deviceId = "MyCSharpDevice";
```

Comment:
we use the deviceId in the SDK samples for all languages, (C, Python, Node), as well as in the Happy Path.